appeals/web/src/server/appeals/appeal-details/accordions/index.js

```diff
@@ -10,13 +10,17 @@ import { FEATURE_FLAG_NAMES } from '@pins/appeals/constants/common.js';
  * @param {{appeal: MappedInstructions}} mappedData
  * @param {import('express-session').Session & Partial<import('express-session').SessionData>} session
  * @param {boolean} [ipCommentsAwaitingReview]
+ * @param {boolean} [appellantFinalCommentsAwaitingReview]
+ * @param {boolean} [lpaFinalCommentsAwaitingReview]
  * @returns {PageComponent}
  */
 export function generateAccordionItems(
 	appealDetails,
 	mappedData,
 	session,
-	ipCommentsAwaitingReview
+	ipCommentsAwaitingReview,
+	appellantFinalCommentsAwaitingReview,
+	lpaFinalCommentsAwaitingReview
```

Comment:
Maybe worth making an object for these? We're gunna have to pass them through for appellant and lpa statements to, and whatever else comes down the line

Reply:
Good idea, thanks! I'll make this update, but I also opened a tech debt ticket to eventually move the notification banner mapping code out of the accordion mapper: https://pins-ds.atlassian.net/browse/A2-1903

Reply:
I've updated this to an object with the new type `RepresentationTypesAwaitingReview` now